### ISSUE TITLE
[Bug]: Empty Text Response Bug in Chat View

### ISSUE THREAD

**Issue by superiorsd10** [Welltested-AI/fluttergpt#141]
### What version are you using?

I'm using `FlutterGPT: AI Copilot with Gemini inside Dart v0.2.0`

### What happened?

When using the chat view in the FlutterGPT VSCode extension, I observed a bug where it responds even when the user enters empty text (whitespace) and presses Enter. This behavior is unexpected, and I believe it would be better to prompt the user to enter something meaningful before pressing Enter when the user doesn't type anything and presses Enter.

**Expected Behavior**

I expect that it should not respond when the user enters empty text, and instead, a prompt or notification (e.g., snack bar) should be displayed, indicating that the user needs to type something before pressing Enter.

### Steps to reproduce

```bash
1. Open the chat view in the FlutterGPT extension.
2. Enter whitespace or leave the text input empty.
3. Press Enter.
```

### Supporting info to reproduce

I'm attaching an image depicting the above-described behavior

<img src="https://github.com/Welltested-AI/fluttergpt/assets/92971894/b03962f4-e644-408a-8828-bf2fc7309cf9" height="700" width="270">

**Comment by superiorsd10** [comment]
Hello @yatendra2001 👋
I want to work on this issue, so please consider assigning it to me.

**Comment by yatendra2001** [comment]
hey @superiorsd10 , thanks for opening this bug. Assigned to you.

**Comment by superiorsd10** [comment]
Hello @yatendra2001 👋
Thanks for assigning this issue to me. I have submitted a [PR](https://github.com/Welltested-AI/fluttergpt/pull/144) for your review.

Thank you,

**Comment by yatendra2001** [comment]
closing this issue, resolved by PR #144
